object/service.proto

```diff
@@ -595,6 +538,81 @@ message SearchResponse {
   neo.fs.v2.session.ResponseVerificationHeader verify_header = 3;
 }
 
+// Object SearchV2 request
+message SearchV2Request {
+  // Object Search request body
+  message Body {
+    // Container where the search is being performed.
+    neo.fs.v2.refs.ContainerID container_id = 1;
+
+    // Version of the Query Language used.
+    uint32 version = 2;
+
+    // List of search expressions. Limited to 8.
+    repeated SearchFilter filters = 3;
+
+    // Cursor to continue search. Can be omitted or empty for the new search.
+    string cursor = 4;
+
+    // Limits the number of responses to the specified number. Can't be more
+    // than 1000.
+    uint32 count = 5;
+
+    // List of attribute names (including special ones as defined by
+    // SearchFilter key) to include into the reply. Limited to 4, these
+    // attributes also affect result ordering (result is ordered by attributes
```

Comment:
just to be sure: only the first one matters, right? `Result is ordered by the first attribute used in filters` but there is "attributes"

Reply:
No, all of them matter.

Reply:
got PR wrong, now it is ok